codec-http2/src/main/java/io/netty/handler/codec/http2/Http2RemoteFlowController.java

```diff
@@ -53,6 +53,11 @@
 
         /**
          * Signal an error and release any retained buffers.
+         *<p>
+         * This method might be called multiple times.
```

Comment:
@nmittler it seems like that right now the `error()` method is usually called at least twice.
When an error happens within the flow controller we invoke `error` directly, which then fails the `ChannelPromise` which has a listener attached that calls `error` in case the promise failed :-)).